dotnetv3/Glue/Actions/GlueWrapper.cs

```diff
@@ -0,0 +1,361 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[Glue.dotnetv3.GlueWrapper]
+
+using System.Net;
+
+namespace GlueActions;
+
+public class GlueWrapper
+{
+    private readonly IAmazonGlue _amazonGlue;
+
+    /// <summary>
+    /// Constructor for the Glue actions wrapper.
+    /// </summary>
+    /// <param name="amazonGlue"></param>
+    public GlueWrapper(IAmazonGlue amazonGlue)
+    {
+        _amazonGlue = amazonGlue;
+    }
+
+    // snippet-start:[Glue.dotnetv3.CreateCrawler]
+    /// <summary>
+    /// Create an AWS Glue crawler.
+    /// </summary>
+    /// <param name="crawlerName">The name for the crawler.</param>
+    /// <param name="crawlerDescription">A description of the crawler.</param>
+    /// <param name="role">The AWS Identity and Access Management (IAM) role to
+    /// be assumed by the crawler.</param>
+    /// <param name="schedule">The schedule on which the crawler will be executed.</param>
+    /// <param name="s3Path">The path to the Amazin Simple Storage Service (S3)
```

Comment:
the Amazin Simple Storage Service (S3) --> the Amazon Simple Storage Service (Amazon S3) bucket

Reply:
fixed